File: output/gemma4-e4b/avatar/Java/Python/atcoder_ABC112_A.py
import sys

def main():
    # Read all input lines
    lines = sys.stdin.read().splitlines()
    
    if not lines:
        return

    try:
        # Read N from the first line
        n = int(lines[0])
    except ValueError:
        return

    if n == 1:
        print("Hello World")
    else:
        # If N != 1, we expect two more numbers on the next two lines
        if len(lines) >= 3:
            try:
                # Read the two numbers
                a = int(lines[1])
                b = int(lines[2])
                print(a + b)
            except ValueError:
                # Handle case where subsequent lines are not integers
                pass

if __name__ == "__main__":
    main()

File: output/gemma4-e4b/avatar/Python/Java/atcoder_ABC112_A.java
import java.util.Scanner;

public class atcoder_ABC112_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read age
        int age = scanner.nextInt();
        
        if (age == 1) {
            System.out.println("Hello World");
        } else {
            // Read a and b
            int a = scanner.nextInt();
            int b = scanner.nextInt();
            
            System.out.println(a + b);
        }
        
        scanner.close();
    }
}